src/views/preview/preview.jsx

```diff
@@ -289,28 +262,6 @@ class Preview extends React.Component {
             loveCount: loves
         });
     }
-    isShared () {
```

Comment:
Thanks for moving these!

Reply:
Sometimes the code just...
![smell bad](https://user-images.githubusercontent.com/3431616/45693981-48d9f980-bb2c-11e8-8056-b8a98a923c60.gif)